sherpa/utils/src/_psf.cc

```diff
@@ -250,6 +259,18 @@ static int _convolve( tcdPyData* self, double* source, double* kernel,
   return EXIT_SUCCESS;
 }
 
+static bool same_size_arrays( npy_intp size1, npy_intp size2,
+                              const char *suffix, const char *name1,
+                              const char *name2 ) {
+  if ( size1 != size2 ) {
+    std::ostringstream err;
+    err << "input array sizes do not match" << suffix;
+    err << name1 << ": " << size1 << name2 << ": " << size2;
```

Comment:
it looks like the 13 calls to this function all pass in " vs ..." as parameter name2. The " vs " segment should be part of the format string here and the name2 should be restricted to the column name